fsh/input/fsh/DentalCare-Bundle-Scenario-3.fsh

```diff
@@ -231,4 +243,53 @@ Usage: #example
     * extension[http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-houseNumber].valueString = "100"
   * city = "Nijmegen"
   * postalCode = "6532 SZ"
-* managingOrganization = Reference(DentalCare-Organization-Defensie-Tandheelkundige-Dienst) "Ministerie van Defensie DTD Tandheelkunde"
\ No newline at end of file
+* managingOrganization = Reference(DentalCare-Organization-Defensie-Tandheelkundige-Dienst) "Ministerie van Defensie DTD Tandheelkunde"
+
+Instance: DentalCare-Payer-Van-De-Stok
+InstanceOf: http://nictiz.nl/fhir/StructureDefinition/nl-core-Payer.InsuranceCompany
+Usage: #example
+* text
+  * status = #generated
+  * div = "<div xmlns='http://www.w3.org/1999/xhtml'>Verzekering: A.S.R<br/>Patiënt: Berend van de Stok<br/>begin datum: 2025-01-01<br/>Status: Actief<br/>Betaler: A.S.R.</div>"
+* status = #active
+* type = $CoverageTypeCodeSystemOID#T "Tandverzekering (los)"
+* subscriberId = "12345679"
+* beneficiary = Reference(DentalCare-Patient-Van-De-Stok) "Berend van de Stok"
+* period
+  * start = "2025-01-01"
+  * end = "2026-01-01"
+* payor = Reference(DentalCare-Organization-ASR) "A.S.R."
+
+Instance: DentalCare-Organization-ASR
+InstanceOf: http://nictiz.nl/fhir/StructureDefinition/nl-core-Payer-Organization
+Usage: #example
+* text
+  * status = #generated
+  * div = "<div xmlns='http://www.w3.org/1999/xhtml'>Organisatie: A.S.R.<br/>Adres: Achillesstraat 12, 1234 AA Amsterdam</div>"
+* name = "A.S.R."
+* address
```

Comment:
Voor A.S.R. moet een ander adres gekozen worden dan voor de Patient (ook aanpassen in narrative van deze resource).

Verder zijn het gestructureerde adres en het adres in de narrative nu niet gelijk aan elkaar.